src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,112 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("fujitsu");
+        user.setAge(33);
+        user.setPassword("RollingInTheDeep");
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void userLoginIsNull_NotOk() {
```

Comment:
https://github.com/mate-academy/jv-registration-validation#:~:text=For%20this%20task%20use%20such%20convention%3A%20%3CmethodUnderTest%3E_%3Cstate%3E_%3CexpectedBehavior%3E%3B%20For%20example%2C%20if%20we%20are%20testing%20the%20method%20register%20with%20a%20null%0Auser%27s%20age%20the%20test%20method%20name%20should%20be%20register_nullAge_notOk.%20notOk%20is%20because%20the%20test%20expects%20the%20register%20method%20to%20throw%20an%20exception.